lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -416,6 +416,8 @@ class PatternStopContents extends Component<Props, State> {
   // eslint-disable-next-line complexity
   render () {
     const {active, activePattern, feedSource, patternEdited, patternStop, tables} = this.props
+
+    if (!this.state) return null
```

Comment:
This looks bad. Better initialize the state field in the class constructor and check individual state fields instead.

Reply:
This is residue from when this was being developed. It looks like it can be safely removed.